anndata/tests/test_views.py

```diff
@@ -565,6 +565,15 @@ def test_negative_scalar_index(adata, index: int, obs: bool):
     )
 
 
+def test_viewness_propagation():
```

Comment:
```suggestion
def test_viewness_propagation():
    # https://github.com/scverse/anndata/issues/239
```

It's good to link the relevant issue for tests so we can go back for context.